heartbeat/azure-lb

```diff
@@ -15,10 +15,9 @@
 # Initialization:
 : ${OCF_FUNCTIONS_DIR=${OCF_ROOT}/lib/heartbeat}
 . ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
-
```

Comment:
Can you re-add this empty line for readability, and squash the commits into a single commit?